### DISCUSSION TITLE
Define contribution process

### DISCUSSION THREAD

**Discussion by beatrizmilhomem** [vtex/shoreline#733] [General]
* Review issues templates
* Define how we'll deal with the issues
* Announce to design and frontend chapters

**Comment by davicostalf** [comment]
Comentários do CSAT de engenharia 2023 H1:

> Ter migrado para as issues não foi um bom caminho na minha opinião. A visibilidade dos problemas é reduzida, é mais difícil identificar problemas que outras pessoas tbm estão tendo em comum, burocratizou um pouco o processo. 
> 
> Outra coisa que acabou acontecendo foi a desistencia em reportar problemas, pois existe uma demora pra corrigir e muitas vezes nem é corrigido, fica mais facil só tentar resolver por conta própria mesmo. 
> 
> Sei que é algo que tira visibilidade do time de voces sobre a real situacao mas realmente era comum algumas coisas ficarem esquecidas e despriorizadas apenas com um promessa de vai ser feito

> Já tive uma experiência de pedir supor para o tipo do Admin UI e foi bem burocrático e no final não tivemos a ajuda que precisamos. Tivemos que fazer um work-around.
